src/main/scala/dspjunctions/JtagAxi.scala

```diff
@@ -0,0 +1,126 @@
+// See LICENSE for license details.
+
+package dspjunctions
+
+import cde._
+import chisel3._
+import chisel3.util._
+import jtag._
+import junctions._
+
+case object DspChainIncludeJtag extends Field[Bool]
+
+class DspJtagIO extends JtagIO {
+  // val TRST = Input(Bool())
+}
```

Comment:
Unfortunately the JTAG base doesn't have async reset support, because Chisel doesn't have async reset support.
I guess we could blackbox it though if needed.